test/unit/spec/util/jobLogViewerSpec.js

```diff
@@ -15,9 +16,15 @@ define([
     const endStates = ['completed', 'error', 'terminated'];
     const queueStates = ['created', 'estimating', 'queued'];
 
-    function getWidgetState(node) {
-        const stateNode = node.querySelector('[data-element="widget-state"]');
-        return JSON.parse(stateNode.textContent);
```

Comment:
the widget state can be assessed by looking at `this.jobLogViewerInstance.state`, so this workaround is no longer needed